src/oca_github_bot/commands.py

```diff
@@ -53,24 +66,22 @@ def delay(self, org, repo, pr, username, dry_run=False):
 
 
 class BotCommandMerge(BotCommand):
-    bumpversion_mode = None  # optional str: major|minor|patch
+    bumpversion_mode = None
+    bumpversion_mode_list = ["major", "minor", "patch", "nobump"]
 
     def parse_options(self, options):
         if not options:
-            return
-        if len(options) == 1 and options[0] in ("major", "minor", "patch"):
+            raise RequiredOptionError(
+                self.name, "bumpversion_mode", self.bumpversion_mode_list
+            )
+        if len(options) == 1 and options[0] in self.bumpversion_mode_list:
             self.bumpversion_mode = options[0]
```

Comment:
Nitpick I would have done this to avoid changing the rest of the code. No big deal though.

```suggestion
            if options[0] != 'nobump':
                self.bumpversion_mode = options[0]
```

Reply:
Yes I get it. You're right, diff could be lighter. 
The problem I see with such implementation is for developpers. IMO,

* if option is optional, the developer expect that bumpversion_mode is in ``["major", "minor", "patch", None]``
* if option is required, the developer expects that bumpversion_mode is in ``["major", "minor", "patch", "nobump"]``

I so implemented that point with such design. If it's a blocking point, I revert, no problem.